src/card/Card.tsx

```diff
@@ -131,6 +134,7 @@ const NeoCard = ({
             database={database}
             active={active}
             setActive={setActive}
+            onDownloadImage={() => downloadComponentAsImage(ref)}
```

Comment:
@alfredorubin96 the download as image seems to be broken now when a report is viewed in fullscreen, could you check?

Reply:
works! now two more things on my mind (we can also decide to fix later):
1. for specific types of charts (e.g. GraphChart) the 'save as image' gives a strange extra margin at the bottom (non-fullscreen)
2. in the fullscreen, it takes also the margins of the window into the screenshot, it would be better if that's not in there.

What do you think? Fix later?